airflow/providers/google/firebase/hooks/firestore.py

```diff
@@ -0,0 +1,142 @@
+
+        :return: Google Cloud Build services object.
+        """
+        if not self._conn:
+            http_authorized = self._authorize()
+            # We cannot use an Authorized Client to retrieve discovery document due to an error in the API.
+            # When the authorized customer will send a request to the address below
+            # https://www.googleapis.com/discovery/v1/apis/firestore/v1/rest
+            # then it will get the message below:
+            # > Request contains an invalid argument.
+            # At the same time, the Non-Authorized Client has no problems.
+            non_authorized_conn = build("firestore", self.api_version, cache_discovery=False)
+            self._conn = build_from_document(
+                non_authorized_conn._rootDesc,  # pylint: disable=protected-access
+                http=http_authorized
+            )
+        return self._conn
+
+    @CloudBaseHook.fallback_to_default_project_id
+    def export_documents(
+        self, body: Dict, database_id: str = "(default)", project_id: Optional[str] = None
+    ) -> None:
+        """
+        Starts a build with the specified configuration.
+
+        :param database_id: The Database ID.
+        :type database_id: str
+        :param body: The request body.
+            See:
+            https://firebase.google.com/docs/firestore/reference/rest/v1beta1/projects.databases/exportDocuments
+        :type body: dict
+        :param project_id: Optional, Google Cloud Project project_id where the function belongs.
+            If set to None or missing, the default project_id from the GCP connection is used.
+        :type project_id: str
+        """
+        if not project_id:
+            raise ValueError("The project_id should be set")
+        service = self.get_conn()
+
+        name = f"projects/{project_id}/databases/{database_id}"
+
+        operation = (
+            service.projects()  # pylint: disable=no-member
+            .databases()
+            .exportDocuments(name=name, body=body)
+            .execute(num_retries=self.num_retries)
+        )
+
+        self._wait_for_operation_to_complete(operation["name"])
+
+    def _wait_for_operation_to_complete(self, operation_name: str) -> None:
+        """
+        Waits for the named operation to complete - checks status of the
+        asynchronous call.
+
+        :param operation_name: The name of the operation.
+        :type operation_name: str
+        :return: The response returned by the operation.
+        :rtype: dict
+        :exception: AirflowException in case error is returned.
+        """
+        service = self.get_conn()
+        while True:
+            operation_response = (
+                service.projects()  # pylint: disable=no-member
+                .databases()
+                .operations()
+                .get(name=operation_name)
+                .execute(num_retries=self.num_retries)
+            )
+            if operation_response.get("done"):
```

Comment:
This is ok, however, I wonder isn't `if "done" in operation_response` more pythonic?

Reply:
We want to check if the done value is True.